test/Hydration-test.js

```diff
@@ -0,0 +1,111 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+const Hydration = require('../src/Hydration.js');
+
+describe('Hydration', function() {
+  let hydrationData;
+
+  beforeEach(function() {
+    hydrationData = [
+      {
+        "userID": 1,
+        "date": "2019/06/15",
+        "numOunces": 37
+      },
+      {
+        "userID": 2,
+        "date": "2019/06/15",
+        "numOunces": 75
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/16",
+        "numOunces": 69
+      },
+      {
+        "userID": 2,
+        "date": "2019/06/16",
+        "numOunces": 91
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/17",
+        "numOunces": 96
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/18",
+        "numOunces": 100
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/19",
+        "numOunces": 20
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/20",
+        "numOunces": 31
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/21",
+        "numOunces": 91
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/22",
+        "numOunces": 55
+      },
+      {
+        "userID": 1,
+        "date": "2019/06/23",
+        "numOunces": 74
+      }
+    ];
+
+    userOneHydration = new Hydration(hydrationData, 1);
```

Comment:
constructor parameter will include all hydration data as well as user id, to distinguish which hydration data will be filtered

Reply:
This works if we're using the `HydrationRepository` class, but I was thinking instantiating the `Hydration` class in each user the way we did with the `User` in `UserRepo`. But then we could sort them by date and remove the `id` property Idk which is a better idea, but it was my first thought.